src/site/content/en/secure/cross-origin-resource-sharing/index.md

```diff
@@ -83,7 +83,7 @@ please."
 
 ### body
 
-The message itself. This could be plain text, an image binary, JSON, HTML,etc.
+The message itself. This could be plain text, an image binary, JSON, HTML, etc.
```

Comment:
```suggestion
The message itself. This could be plain text, an image binary, JSON, HTML, and so on.
```

Google Developer Documentation Style Guide instructs us to avoid `etc.` https://developers.google.com/style/word-list#letter-e